app/components/work_package_types/pattern_input.html.erb

```diff
@@ -94,7 +94,7 @@ See COPYRIGHT and LICENSE files for more details.
         Primer::BaseComponent.new(
           tag: :div,
           contenteditable: true,
-          border: true, border_radius: 2, p: 1, classes: :input,
+          border: true, border_radius: 2, p: 1, classes: "op-pattern-input--text-field",
```

Comment:
I guess, this is the wrong element to set the styling on.. The styles were written for the very first container div of the dropdown (the sibling of the input div). That is also the reason the shadow and the border is missing that @NobodysNightmare mentioned 
<img width="1078" alt="Bildschirmfoto 2025-07-04 um 13 51 09" src="https://github.com/user-attachments/assets/94e1081f-118f-4b0e-abd6-74350dbbb5f4" />